### PR TITLE
Add option to open browser tabs with bug reports for unfiled bugs

### PR DESCRIPTION
**Pull request by encukou**
I actually find this more convenient than auto-filing the bugs.
It gives me a chance to verify and/or adjust the message before filing a bug in my name.

### CODE REVIEW

**Comment by encukou** [comment]
cc @hroncok

File: monitor_check.py [hunk 1]
```diff
@@ -127,8 +131,49 @@ def p(*args, **kwargs):
...
+    description = dedent(f"""
+        {package} fails to build with Python 3.8.0b1.
+
+        This report is automated and not very verbose, but we'll try to get back here with details.
+
+        For the build logs, see:
+        https://copr-be.cloud.fedoraproject.org/results/@python/python3.8/fedora-rawhide-x86_64/{build:08}-{package}/
+
+        For all our attempts to build {package} with Python 3.8, see:
+        https://copr.fedorainfracloud.org/coprs/g/python/python3.8/package/{package}/
```
Comment (hroncok) [inline monitor_check.py:150]: Testing and mass rebuild of packages is happening in copr. You can follow these instructions to test locally in mock if your package builds with Python 3.8: https://copr.fedorainfracloud.org/coprs/g/python/python3.8/

File: monitor_check.py [hunk 2]
```diff
@@ -127,8 +131,49 @@ def p(*args, **kwargs):
...
+    """)
+
+    url_prefix = 'https://bugzilla.redhat.com/enter_bug.cgi?'
+    params = {
+        'short_desc': summary,
+        'comment': description,
+        'component': str(package),
+        'dependson': 'PYTHON38',
+        'product': 'Fedora',
+        'version': 'rawhide',
```
Comment (hroncok) [inline monitor_check.py:165]: severity high?

**Review by hroncok — APPROVED** [review]

File: monitor_check.py [hunk 3]
```diff
@@ -127,8 +131,55 @@ def p(*args, **kwargs):
         message += ' \N{FIRE}'
     p(message, fg=fg)
 
+    if browser_lock and not bz and content_length > LIMIT:
```
Comment (hroncok) [inline monitor_check.py:134]: or bz is closed (see the logic for fg = red)

File: monitor_check.py [hunk 4]
```diff
@@ -127,8 +131,55 @@ def p(*args, **kwargs):
...
+
+        Let us know here if you have any questions.
+    """)
+
+    url_prefix = 'https://bugzilla.redhat.com/enter_bug.cgi?'
+    params = {
+        'short_desc': summary,
+        'comment': description,
+        'component': str(package),
+        'dependson': 'PYTHON38',
```
Comment (hroncok) [inline monitor_check.py]: this dependency shall be reversed

**Review by hroncok — CHANGES_REQUESTED** [review]

**Pull request merged**